perseo-main/src/main/java/com/telefonica/iot/perseo/Configuration.java

```diff
@@ -67,7 +69,7 @@ public static synchronized boolean reload() {
         InputStream stream;
         String defaultMaxAge;
         String defaultURL;
-        String actionPath = "/actions/do";
+        String actionPath = ResourceBundle.getBundle("perseo-core").getString("action.path");
```

Comment:
Not sure if this is correct...

I mean, looking above to:

```
     private static final String PATH = ResourceBundle.getBundle("perseo-core").getString("properties.path");
```

I understand that `ResourceBundle.getBundle("...").getString("...")` is used to get the path of a *filename*. However, in case, `/actions/do` doesn't refer to a filename, but to an URL endpoint...

Reply:
Looking to the content of perseo-main/src/main/resources/perseo-core.properties it seems I misunderstand this. Thus `ResourceBundle.getBundle("...").getString("...")` seems to be a way of getting properties from that file. Is my understanding correct?

Reply:
The change is due to a rule that detects hardcoded url. If you consider that the path is not a url we reverse the change.

Reply:
Please find my answer here: https://github.com/telefonicaid/perseo-core/pull/151#discussion_r371376184

Reply:
Fixed in 47ab83625cbfba55660da30a785723e2fd7ecf55

Reply:
Travis ci fixed in b4a4d739cc56604002ce38463fa15aeccb64b823